Gemfile.lock

```diff
@@ -257,6 +263,7 @@ GEM
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229280624-9ba4212b-a3b0-4034-9e11-4f4e7e3ff092.png)

- [ ] It would be nice if you follow the Gitflow for the submission of your project because now you are merging your feature branch into your master branch 👍 but you are supposed to merge your feature branch to development branch since you are required to follow the Gitflow 😊

Reply:
I did that in that in the last code review and he told me it's invailed